packages/framework/src/schemas/providers/email/index.ts

```diff
@@ -1,19 +1,15 @@
 import { EmailProviderIdEnum } from '@novu/shared';
 import { Schema } from '../../../types';
 import { genericProviderSchemas } from '../generic.schema';
-import { mailgunProviderSchemas } from './mailgun.schema';
-import { mailjetProviderSchemas } from './mailjet.schema';
-import { nodemailerProviderSchemas } from './nodemailer.schema';
 import { novuEmailProviderSchemas } from './novu-email.schema';
-import { sendgridProviderSchemas } from './sendgrid.schema';
 
 export const emailProviderSchemas = {
-  [EmailProviderIdEnum.SendGrid]: sendgridProviderSchemas,
+  [EmailProviderIdEnum.SendGrid]: genericProviderSchemas,
```

Comment:
This interface is going to change next week, so we are reverting all Providers to the generic schema for now.